scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -45,15 +49,63 @@ public static BuildTargetResult InitPublish(BuildTargetContext c)
         [Environment("PUBLISH_TO_AZURE_BLOB", "1", "true")] // This is set by CI systems
         public static BuildTargetResult Publish(BuildTargetContext c)
         {
+            if (ShouldPublishToLatest())
```

Comment:
If the last two builds run the publish step at the same time, seems like this will break down.